model/model.go

```diff
@@ -77,6 +77,7 @@ type SystemInstall struct {
 	CryptPass         string                 `yaml:"-"`
 	MakeISO           bool                   `yaml:"iso,omitempty,flow"`
 	KeepImage         bool                   `yaml:"keepImage,omitempty,flow"`
+	SystemCheck       bool                   `yaml:"systemCheck,omitempty,flow"`
```

Comment:
Maybe we should drop this from the configuration file for now since we only support running this from the command line standard alone?

Reply:
Done!